rule_match.go

```diff
@@ -58,23 +62,26 @@ type MatchedRule struct {
 	// Client IP address
 	ClientIPAddress string
 	// A slice of matched variables
+	MatchedDatas []MatchData
+	// Deprecated
```

Comment:
If there is no place to use, you can delete it, thanks.

Reply:
Juan advised against it. As someone already using it could cause errors.
We can remove it in next release.

Reply:
@bxlxx we should mark unused exported variables as deprecated before removing them in the next major release.